docs/local_install_skaffold.md

```diff
@@ -108,81 +325,287 @@ Running the following command will do it for you (you'll be asked for your passw
 echo "192.168.65.2 substra-backend.node-1.com substra-frontend.node-1.com substra-backend.node-2.com substra-frontend.node-2.com" | sudo tee -a /etc/hosts
 ```
 
-### 4. Install skaffold
+## 3. Running the platform
 
-On OSX, if you have homebrew installed, just run
+### Start Substra
 
-```
-brew install skaffold
-```
+- For Ubuntu, once Minikube is running and Tiller initialized, please go the `hlf-k8s`, `substra-backend` and `substra-frontend` repositories and run `skaffold dev` to start Substra with Skaffold.
+- For Mac, directly head to the tree repositories and start Substra with Skaffold.
```

Comment:
I think we could avoid the distinction between Ubuntu and Mac here: both setups require a Kubernetes cluster up and running (started via Minikube or Docker for mac) and Tiller initialized.

Reply:
Done, thank you!